### PR TITLE
MTDSA-221 - Calculate Total Profit from Furnished Holiday Lettings.

### PR DESCRIPTION
**Pull request by hicnar**

### CODE REVIEW

File: app/uk/gov/hmrc/selfassessmentapi/domain/Liability.scala [hunk 1]
```diff
@@ -160,6 +163,17 @@ object Liability {
     }
   }
 
+  private def exampleFurnishedHolidayLettingIncomes = {
+    FeatureSwitch(AppContext.featureSwitch).isEnabled(SourceTypes.FurnishedHolidayLettings) match {
+      case true =>
+        Seq(
+          FurnishedHolidayLettingIncome("furnished-holiday-letting-1", 8200),
```
Comment (danmassie) [inline app/uk/gov/hmrc/selfassessmentapi/domain/Liability.scala]: With the inclusion of additional random values, the example liability JSON that's included in the documentation doesn't make sense as the output will be incorrect. The liability example should be a valid calculation output as we know that those API consumers that understand SA do inspect the values and would expect to see an output that makes sense. We are relying on these values for tests and docs which IMO isn't a good fit.

File: test/uk/gov/hmrc/selfassessmentapi/services/live/calculation/steps/FurnishedHolidayLettingsProfitCalculationSpec.scala [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+package uk.gov.hmrc.selfassessmentapi.services.live.calculation.steps
+
+import uk.gov.hmrc.selfassessmentapi.domain.furnishedholidaylettings.ExpenseType._
+import uk.gov.hmrc.selfassessmentapi.domain.furnishedholidaylettings.{Adjustments, Allowances}
+import uk.gov.hmrc.selfassessmentapi.repositories.domain._
+import uk.gov.hmrc.selfassessmentapi.{FurnishedHolidayLettingsSugar, UnitSpec}
+
+class FurnishedHolidayLettingsProfitCalculationSpec extends UnitSpec with FurnishedHolidayLettingsSugar {
+
+  private val liability = MongoLiability.create(generateSaUtr(), taxYear)
```
Comment (GlassAndOneHalf) [inline test/uk/gov/hmrc/selfassessmentapi/services/live/calculation/steps/FurnishedHolidayLettingsProfitCalculationSpec.scala]: If you mix in `UnitSpecSugar` you can use the `aLiability` method here.

File: app/uk/gov/hmrc/selfassessmentapi/domain/Liability.scala (hunk 1 above)
Comment (hicnar) [inline app/uk/gov/hmrc/selfassessmentapi/domain/Liability.scala]: Agreed after a short discussion. Looks like this needs to be discussed further and addressed in a separate story.

File: test/uk/gov/hmrc/selfassessmentapi/services/live/calculation/steps/FurnishedHolidayLettingsProfitCalculationSpec.scala (hunk 2 above)
Comment (hicnar) [inline test/uk/gov/hmrc/selfassessmentapi/services/live/calculation/steps/FurnishedHolidayLettingsProfitCalculationSpec.scala]: Applied

**Comment by danmassie** [comment]
:+1:

**Pull request merged**